### PR TITLE
Flock integration

### PR DESCRIPTION
**Pull request by mx4492**
Add a action that allows sending text messages to Flock groups
(http://flock.co).

### CODE REVIEW

File: fastlane/lib/fastlane/actions/flock.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    class FlockAction < Action
+      BASE_URL = 'https://api.flock.co/hooks/sendMessage'.freeze
+
+      def self.run(options)
+        require 'net/http'
+        require 'uri'
+        require 'json'
+
+        notify_incoming_message_webhook(
+          options[:base_url], options[:message], options[:token])
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/flock.rb]: Could this be in just one line?

File: fastlane/lib/fastlane/actions/flock.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        if response.kind_of? Net::HTTPSuccess
+          UI.success 'Message sent to Flock.'
+        else
+          UI.error "HTTP request to '#{uri}' with message '#{message}' failed with a #{response.code} response."
+          UI.user_error! 'Error sending message to Flock. Please verify the Flock webhook token.'
+        end
+      end
+
+      def self.description
+        "Send a message to a Flock group [http://flock.co]"
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/flock.rb]: Please remove the URL here

**Comment by KrauseFx** [comment]
This looks great, also thanks for adding so many tests :+1: I added some comments before it's ready to be merged

**Comment by mx4492** [comment]
Updated

**Comment by asfalcone** [comment]
Thanks @mx4492  :+1:

**Pull request merged**